openapi.yml

```diff
@@ -357,6 +353,19 @@ components:
       x-extensible-enum:
         - "LOCAL"
         - "NATIONAL"
+    RequestStatus:
+      type: string
+      x-extensible-enum:
+        - "CREATED"
```

Comment:
please explain these states in the api docs above. what's the difference between created and submitted ?